docs/university/course-four-workshop.md

```diff
@@ -0,0 +1,349 @@
+
+Next, let’s set our block to be “Historical” and the duration and resolution to “15 minutes” and “1 minute” respectively.
+
+![Time Series Graph Settings](/images/university/course-four-workshop/time-series-settings.png)
+
+The simulator is set to generate data every two seconds. So, by choosing a resolution of one minute, each data point on the block will represent an aggregated value of 30 data points (data will be reported 30 times, or every two seconds, within a one minute interval). We will have the option to select the aggregation method below.
+
+Next, configure the device data.
+
+![Time Series Graph Block Data Settings](/images/university/course-four-workshop/time-series-settings-two.png)
+
+1. Choose the “Water Pump” device.
+2. Select an attribute. In the screenshot above, I chose flow.
+3. Set the “Series Label” to “Flow”.
+4. Set the aggregation to “MEAN”.
+
+Because we chose “MEAN”, each data point on the graph will represent the average of the 30 points we talked about previously.
+
+Now, we should see data appearing in the preview:
+
+![Time Series Graph Preview](/images/university/course-four-workshop/time-series-preview.png)
+
+### Time Series Graph Exercise:
+
+We wanted this block to display voltage, flow and ​temperature. Using the “Add Segment” button, add voltage and ​temperature to this graph.
+
+Once done, you should have a block that looks similar to this:
+
+![Completed Time Series Graph](/images/university/course-four-workshop/time-series-complete.png)
+
+After completing the exercise, add the block to the dashboard.
+
+Next, let’s add a Gauge Block to display the RPM of the water pump.
+
+### Adding a Gauge Block
+
+On the “Add Block” page, click the Gauge Block:
+
+![Gauge Block Add](/images/university/course-four-workshop/gauge-block.png)
+
+First, we must give our block a name and choose our application.
+
+Since we are concerned with it’s current value within a range, this is a perfect use case for the “Live Stream” type. With this type, the block on the dashboard will just reflect a live stream of data coming in.
+
+For our Gauge type, let’s choose a Dial Gauge, a min of 2000, and a max of 3000. These numbers are the same min and max we choose in the simulator.
+
+![Gauge Settings](/images/university/course-four-workshop/gauge-settings.png)
+
+Now, on to our block data:
+
+![Gauge Block Data Settings](/images/university/course-four-workshop/gauge-data-settings.png)
+
+1. Choose “Water Pump” as your device.
+2. Set “Label” to “RPM”.
+3. Choose the “rpm” attribute.
+
+Once done, our preview should appear:
+
+![Gauge Block Preview](/images/university/course-four-workshop/gauge-preview.png)
+
+### Gauge Block Exercise:
+
+The Gauge block can change colors based on its current represented value. Let’s say, for our water pump, the rpm should never reach above 2500 rpm. If it does, our dashboard should reflect that, like so:
```

Comment:
"2500 rpm" - that really should be "2500 RPM" since at this point you're referring to the actual acronym and not the attribute name.